ui/app/styles/components/overview-card.scss

```diff
@@ -5,7 +5,6 @@
 
 .overview-card {
   padding: $spacing-l;
-  display: initial;
```

Comment:
This was causing weird display issues where the card would crunch and the border would break when used within a flexbox. Removing it didn't seem to change anything with existing usages.

![image](https://github.com/hashicorp/vault/assets/24611656/9795b1a4-ad9a-4e89-8bc2-0cee6356907e)